transfuse/pom.xml

```diff
@@ -53,8 +53,9 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <version>2.4</version>
+            <version>2.5</version>
```

Comment:
:+1:... there are a few more commons-io used in testing, could you upgrade those as well in this pass?

Reply:
done